stubs/retry/@tests/stubtest_allowlist.txt

```diff
@@ -1,6 +1,4 @@
 # TODO: missing from stub
```

Comment:
Oh, I just noticed that this comment is now obsolete. At least the "TODO" part, since the other items seem to be internal stuff.

Reply:
removed